packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -160,6 +191,14 @@ export default class QuanticGeneratedAnswer extends LightningElement {
     return this?.state?.isStreaming;
   }
 
+  get shouldDisplayFeedback() {
+    return this.isVisible && !this.isStreaming;
```

Comment:
```suggestion
    return this.isVisible !== false && !this.isStreaming;
```
Maybe if we are specific to expect false?

Reply:
I think it's clear enough to have a simple condition as `if (this.isVisible`)